python/test/test_user_transform.py

```diff
@@ -102,6 +102,4 @@ def test_user_constructors():
     print(meas(2))
     print(meas.map(1))
 
-    post = dp.new_function(lambda x: x[0], dp.i32)
-
-    print((meas >> post)(2))
+    print((meas >> (lambda x: x[0]))(2))
```

Comment:
Not really about this change -- It looks like a good confirmation that we can chain a UDF -- but how useful do you find the print statements in tests? Do you look at stdout sometimes on test runs? If you have expectations about what we should see, would it make sense to add notes to the print statements?

Testing the PCA, I think you mentioned giving it a ridiculously large epsilon, so you could actually make an assertion on the output. Is that something we could do in other places?

Reply:
Yeah, for noise addition mechanisms you could set scale to zero. There's no good reason why these weren't assertions, they should be assertions.

Reply:
This change in particular increased code coverage to cover the automatic wrapping of postprocessor functions into a postprocessor Function